test/integration-serial/serial_deploy_test.rb

```diff
@@ -503,6 +503,25 @@ def test_apply_failure_with_sensitive_resources_hides_template_content
     refute_logs_match("kind: Deployment") # content of the sensitive template
   end
 
+  def test_global_deploy_task
+    KubernetesDeploy::FormattedLogger.expects(:build).returns(@logger)
+    global_deploy = ::Krane::GlobalDeployTask.new(
+      context: task_config.context,
+      template_paths: [fixture_path('globals')],
+      max_watch_seconds: 300,
+      selector: KubernetesDeploy::LabelSelector.parse('app=krane')
+    )
+    global_deploy.run!(verify_result: true, prune: false)
```

Comment:
This test is logging:
```
I, [2019-10-23T16:46:47.093069 #38172]  INFO -- : [StatsD]  Deployment succeeded:Successfully deployed all resources to minikube #context: #status:success
I, [2019-10-23T16:46:47.093143 #38172]  INFO -- : [StatsD] distribution all_resources.duration:1.1 #context: #status:success
```

I suspect something is configuring statsd different for the global_deploy_task than deploy_task.